search/src/Components/dataset/dataset_summary.js

```diff
@@ -51,6 +51,10 @@ const baseURL = new URL(window.location.origin);
 
 const AgContextFieldList = (props) => {
     const {title, agcontext, fields, classes, ...accordionProps} = props;
+    const formatters = {
+      bbch_growth_range: (val) => (val["min"] ? val["min"] + " to " + val["max"] : val),
```

Comment:
```suggestion
      bbch_growth_range: (val) => (val["min"] && val["max"] ? val["min"] + " to " + val["max"] : val),
```

Reply:
If we trust that it's valid according to the schema, then the value is either "na" or {"min": int, "max": int}:

https://github.com/Sydney-Informatics-Hub/Weed-ID-Interchange/blob/f11e6bb3ca96ed6714d1b13144c995c87c5698ba/weedcoco/schema/AgContext.yaml#L59-L79